vmdb/spec/factories/configuration_profile_foreman.rb

```diff
@@ -0,0 +1 @@
+FactoryGirl.define { factory :configuration_profile_foreman, :class => "ConfigurationProfileForeman", :parent => :configuration_profile }
```

Comment:
We generally keep the child factories in the same file as the parent factory, for the most part, but this is ok.